src/features/callAssignments/components/CallAssignmentStatusCards.tsx

```diff
@@ -150,13 +149,11 @@ const CallAssignmentStatusCards: FC<CallAssignmentStatusCardsProps> = ({
             />
             <StatusCardItem
               action={
-                <Button
-                  onClick={() => getOrganizerActionView()}
-                  startIcon={<Visibility />}
-                  variant="outlined"
-                >
-                  <Msg id={messageIds.blocked.viewSheetButton} />
-                </Button>
+                <NextLink href={`/organize/${orgId}/people/lists/blocked`}>
```

Comment:
I'm thinking about what is a good URL for this list. The `blocked` name will be confusing once people can be blocked for more reasons (e.g. email), so I thin that part of the URL should change to `call_blocked`, or maybe `callblocked` ( to be more consistent with how `callassignments` are spelled in the URLs).

Reply:
will change to `callblocked`!